contracts/schemes/TransitionScheme.sol

```diff
@@ -0,0 +1,88 @@
+pragma solidity 0.5.17;
+
+import "../controller/Controller.sol";
+
+/**
+ * @title A scheme for transitioning the DAO's assets to a new one.
+ */
+
+contract TransitionScheme {
+
+    Avatar public avatar;
+    address payable public newAvatar;
+    address[] public externalTokens;
+    address[] public assetAddresses;
+    bytes4[] public selectors;
+
+    /**
+     * @dev initialize
+     * @param _avatar the avatar to migrate from
+     * @param _newAvatar the avatar to migrate to
+     * @param _externalTokens external tokens to allow transfer to the new avatar
```

Comment:
Why is _externakTokens not just to be included in `_assetAddresses` + `_selectors`?

Reply:
I don't think I understand the question. Tokens contracts are not owned by the DAO, only amounts of tokens, so it's a different function.

Reply:
ok, that answers the question.